src/sentry/mediators/external_requests/alert_rule_action_requester.py

```diff
@@ -0,0 +1,84 @@
+import logging
+from urllib.parse import urlencode, urlparse, urlunparse
+from uuid import uuid4
+
+from sentry.http import safe_urlread
+from sentry.mediators import Mediator, Param
+from sentry.mediators.external_requests.util import send_and_save_sentry_app_request
+from sentry.utils import json
+from sentry.utils.cache import memoize
+
+logger = logging.getLogger("sentry.mediators.external-requests")
+
+
+class AlertRuleActionRequester(Mediator):
+    """
+    Makes a POST request to another service to fetch/update the values for each field in the
+    AlertRuleAction settings schema
+    """
+
+    install = Param("sentry.models.SentryAppInstallation")
+    uri = Param((str,))
+    fields = Param(object, required=False, default={})
+    http_method = Param(str, required=False, default="POST")
+
+    def call(self):
+        return self._make_request()
+
+    def _build_url(self):
+        urlparts = list(urlparse(self.sentry_app.webhook_url))
+        urlparts[2] = self.uri
+        if self.http_method == "GET":
```

Comment:
@AniketDas-Tekky I think we can remove this case, bc we will be saving the schema in Sentry. No need to fetch from the app provider.